src/main.ts

```diff
@@ -83,6 +90,8 @@ export async function main() {
         await deleteFile(scriptFilePath);
         console.log("cleaning up container...");
         await executeDockerCommand(["rm", "--force", CONTAINER_NAME], true);
+        // Reset AZURE_HTTP_USER_AGENT
+        core.exportVariable('AZURE_HTTP_USER_AGENT', prefix);

```

Comment:
According to L14, if `AZURE_HTTP_USER_AGENT` is not set, `prefix` will be set to an empty string `""`. This line will make this action modify `AZURE_HTTP_USER_AGENT` from unset to an empty string.

Reply:
Good point. Will also consider this for login and powershell action.